prometheus/indico_prometheus/metrics.py

```diff
@@ -0,0 +1,149 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2002 - 2023 CERN
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+from datetime import timedelta
+
+from prometheus_client import metrics
+
+from indico.core.db import db
+from indico.modules.attachments.models.attachments import Attachment, AttachmentFile
+from indico.modules.auth.models.identities import Identity
+from indico.modules.categories.models.categories import Category
+from indico.modules.events.models.events import Event
+from indico.modules.rb.models.reservation_occurrences import ReservationOccurrence
+from indico.modules.rb.models.reservations import Reservation
+from indico.modules.rb.models.rooms import Room
+from indico.modules.users.models.users import User
+from indico.util.date_time import now_utc
+
+from indico_prometheus.queries import get_attachment_query, get_note_query
+
+
+# Check for availability of the LiveSync plugin
+LIVESYNC_AVAILABLE = True
+try:
+    from indico_livesync.models.queue import LiveSyncQueueEntry
+except ImportError:
+    LIVESYNC_AVAILABLE = False
```

Comment:
This check is not super reliable, since installing `indico-plugins` will install the livesync plugin as well. But unless you can disable a metric at a later point (not during import time) I don't see a better solution.
But in case you *can* disable a metric later, then I'd check if the livesync plugin is actually loaded.

Reply:
Not sure metrics can be disabled, at least I didn't find any way to do it. Maybe I can check the config at load time? The app should be already set up by now, right?

Reply:
I think accessing the config at import time is dangerous. It MAY work when the plugin is being loaded, but any "normal" import outside an app context would break. Probably better to just keep it as-is...